runtime/permissions/mod.rs

```diff
@@ -1931,6 +1975,15 @@ impl deno_fs::FsPermissions for PermissionsContainer {
     self.0.lock().write.check(path, Some(api_name))
   }
 
+  fn check_write_blind(
```

Comment:
```suggestion
  #[inline(always)]
  fn check_write_blind(
```

Reply:
@bartlomieju We don't do that on any of the trait implementations (this is the `fn check_write_blind` in `impl deno_fs::FsPermissions`).

Reply:
Maybe this a good follow up PR for someone else. Don't want to clutter this PR with too much unrelated stuff.